libqtile/__init__.py

```diff
@@ -20,7 +20,15 @@
 # SOFTWARE.
 
 
-qtile = None
+class UndefinedCore:
+    name = None
+
+
+class UndefinedQtile:
+    core = UndefinedCore()
+
+
```

Comment:
Please could we make these private to the module, assuming we don't need them elsewhere?

Reply:
I'll change this to a single underscore prefix unless you want full on name mangling too.